app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt

```diff
@@ -167,4 +176,10 @@ class MaskActivity : AppCompatActivity(), MaskView, CaptureFlow {
         options.inPreferredConfig = Bitmap.Config.ARGB_8888
         BitmapFactory.decodeFile(filepath, options)
     }
+
+    private suspend fun readMask(filepath: String): Bitmap? = withContext(Dispatchers.IO) {
+        val bitmap = BitmapFactory.decodeFile(filepath)
+        //Converts to a mutable bitmap
+        bitmap.copy(Bitmap.Config.ARGB_8888, true)
```

Comment:
@eduardb  Well `readImage()` returns a `Bitmap` whereas `readMask()` returns a nullable `Bitmap?` (as it may not exist). I didn't wanna introduce nullability on the normal flow just for the editing so I made `readMask()` into a separate method.

As for moving the part where we make it mutable, sure we could do that in the `MaskLayer` rather than the `MaskActivity` if you prefer.